assets/js/modules/idea-hub/components/settings/SettingsView.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * Idea Hub Settings View component
+ *
+ * Site Kit by Google, Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * WordPress dependencies
+ */
+import { __, sprintf } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import Data from 'googlesitekit-data';
+import { STORE_NAME } from '../../datastore/constants';
+import Link from '../../../../components/Link';
+import { createInterpolateElement } from '@wordpress/element';
```

Comment:
This should go within the WordPress dependencies section.

Reply:
https://github.com/google/site-kit-wp/pull/3557/commits/62aa044ad7691b28a13d3f21c7eb95293a245b44